pages/createlisting.js

```diff
@@ -11,401 +11,420 @@ import Button from "@mui/material/Button";
 
-    async function fileHandler(e) {
-        /*Upload image to IPFS */
-        const file = e.target.files[0];
-        setSelectedImage(file);
-        try {
-            const added = await client.add(file, {
-                progress: (prog) => console.log(`received: ${prog}`),
-            });
-            const url = DEDICATED_URL + added.path;
-            client.pin.add(added.path).then((res) => {
-                setFileURL(url);
-            });
-        } catch (error) {
-            console.error("Error uploading file", error);
-        }
+  async function fileHandler(e) {
+    /*Upload image to IPFS */
+    const file = e.target.files[0];
+    setSelectedImage(file);
+    try {
+      const added = await client.add(file, {
+        progress: (prog) => console.log(`received: ${prog}`),
+      });
+      const url = DEDICATED_URL + added.path;
+      client.pin.add(added.path).then((res) => {
+        setFileURL(url);
+      });
+    } catch (error) {
+      console.error("Error uploading file", error);
     }
-    return (
-        <>
-            <input
-                accept="image/*"
-                type="file"
-                id="select-image"
-                style={{ display: "none" }}
-                onChange={fileHandler}
-            />
-            <label htmlFor="select-image">
-                <div className="search-button-box">
-                    <ThemeProvider theme={theme}>
-                        <Button
-                            className="create-button"
-                            variant="contained"
-                            color="primary"
-                            component="span"
-                        >
-                            Upload Image
-                        </Button>
-                    </ThemeProvider>
-                </div>
-            </label>
-            {imageUrl && selectedImage && (
-                <div className="search-button-box" mt={2} textAlign="center">
-                    <img
-                        src={imageUrl}
-                        alt={selectedImage.name}
-                        height="300px"
-                    />
-                </div>
-            )}
-        </>
-    );
+  }
+  return (
+    <>
+      <input
+        accept="image/*"
+        type="file"
+        id="select-image"
+        style={{ display: "none" }}
+        onChange={fileHandler}
+      />
+      <label htmlFor="select-image">
+        <div className="search-button-box">
+          <ThemeProvider theme={theme}>
+            <Button
+              className="create-button"
+              variant="contained"
+              color="primary"
+              component="span"
+            >
+              Upload Image
+            </Button>
+          </ThemeProvider>
+        </div>
+      </label>
+      {imageUrl && selectedImage && (
+        <div className="search-button-box" mt={2} textAlign="center">
+          <img src={imageUrl} alt={selectedImage.name} height="300px" />
```

Comment:
From NextJS documentation: "Using `<img>` could result in slower LCP and higher bandwidth. Use `<Image />` from `next/image` instead to utilize Image Optimization. See: https://nextjs.org/docs/messages/no-img-element"

Reply:
Interesting, I didn't know that. That's good to know